server/tests/steps/test_waterfall.py

```diff
@@ -153,3 +153,58 @@ def test_bug_duplicate_rows():
     )
 
     assert_dataframes_equals(result_df, expected_df)
+
+
+def test_waterfall_bug_drill():
+    base_df = pd.DataFrame(
+        {
+            'grand parent': ['A', '1/A', 'i/1/A'] * 2,
+            'parent': ['1/A', 'i/1/A', 'U/i/1/A'] * 2,
+            'label': ['i/1/A', 'U/i/1/A', 'x/U/i/1/A'] * 2,
```

Comment:
This example is quite complex to undersand due to labels :
Maybe it's me but I think something a bit more concrete would be easier to read:
```suggestion
            'grand parent': ['France', 'IDF', 'Paris'] * 2,
            'parent': ['IDF', 'Paris', 'XXe arr'] * 2,
            'label': ['Paris', 'XXe arr', 'Place des Fêtes'] * 2,
```
or any other well-known hierarchy

Reply:
Actually my sugegstion is very frenchy-centric.
Food > Vegetarian > Fruits > Berries > Blueberries may be easier to understand for everyone :p